bundles/org.openhab.binding.pihole/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,164 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="pihole"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<thing-type id="server">
+		<label>PiHole Binding Thing</label>
+		<description>Sample thing for PiHole Binding</description>
```

Comment:
Please update the default description

Reply:
done

Reply:
Still sample, probably edited at the wrong place?

Reply:
done